src/data/pokemon/species_info.h

```diff
@@ -43106,10 +43322,10 @@ const struct SpeciesInfo gSpeciesInfo[] =
         .height = 4,
         .weight = 50,
         .description = COMPOUND_STRING(
-            "It is said to carry wandering spirits to\n"
-            "the place where they belong so they can\n"
-            "move on. As the sun sets, it becomes\n"
-            "restless and active."),
+            "The light that streams out from\n"
+            "the holes in the pumpkin can\n"
+            "hypnotize and control the people and\n"
+            "Pokémon that see it."),
```

Comment:
Same here. Not sure if this is good or bad. I guess there isn't a rule for what desc. will be used?